packages/forge/blocks/zendeskChat/actions/openMessenger.ts

```diff
@@ -0,0 +1,65 @@
+import { createAction, option } from '@typebot.io/forge'
+import { auth } from '../auth'
+
+export const openMessenger = createAction({
+  auth,
+  name: 'Open Messenger',
+  options: option.object({
+    key: option.string.layout({
+      label: "Web Widget Key",
```

Comment:
Can you avoid capital case for option fields name? This would be better if it is "Web widget key" instead.